packages/playground/src/components/pull-request/hooks/useEmitCodeCommentStatus.ts

```diff
@@ -0,0 +1,45 @@
+import { useCallback, useEffect } from 'react'
+import { CodeCommentState } from '../interfaces'
+
+export const PR_COMMENT_STATUS_CHANGED_EVENT = 'PR_COMMENT_STATUS_CHANGED_EVENT'
+export const PULL_REQUEST_ALL_COMMENTS_ID = -99999
```

Comment:
Is this supposed to be a minimum integer value? Probably should use `Number.MIN_SAFE_INTEGER` in that case.

Reply:
fixed